tests/core/test_time.h

```diff
@@ -46,6 +46,133 @@
 
 namespace TestTime {
 
+TEST_CASE("[Time] Datetime dictionary conversion to/from datetime string") {
```

Comment:
This test case method is quite long, to the point that it may make sense to split it into two test cases (one dict -> string, and one string -> dict).